apps/api/src/Match/Match.service.ts

```diff
@@ -0,0 +1,110 @@
+import { Injectable, BadRequestException } from '@nestjs/common';
+import { InjectRepository } from '@nestjs/typeorm';
+import { TransformUserService } from 'src/TransformUser/TransformUser.service';
+import { Repository } from 'typeorm';
+import { Match, User, Userfront } from 'types';
+import { UsersService } from '../users/users.service';
+
+export interface MatchFront {
```

Comment:
First, you should avoid interface definitions in services for code flexibility reasons. Second, as it is called front, I suppose you use the same interface in the front, thus it should be in the common types package

Reply:
done